istqb-ctal-att-complete-guide.md

```
@@ -37,69 +37,63 @@
-```
-Chapter 1: 要件エンジニアリング          ██████        ~90分
-Chapter 2: アジャイルにおけるテスト       ████████████ ~360分 ← 最重要
-Chapter 3: テスト自動化                  ████████     ~270分
-Chapter 4: デプロイメントとデリバリー     ███████      ~240分
-                                         合計: ~960分 (16時間)
-```
+> Chapter 1: 要件エンジニアリング          ██████        ~90分
+> Chapter 2: アジャイルにおけるテスト       ████████████ ~360分 ← 最重要
+> Chapter 3: テスト自動化                  ████████     ~270分
+> Chapter 4: デプロイメントとデリバリー     ███████      ~240分
+>                                          合計: ~960分 (16時間)
```

markdownlint-cli2 (0.22.0)

[warning] Multiple spaces after blockquote symbol

(MD027, no-multiple-space-blockquote)